cypress/integration/basic-functionalities/local-tests/compose/namespace/Create_a_namespace.spec.js

```diff
@@ -33,9 +34,10 @@ describe('Test for creating a namespace', () => {
     })
   })
 
-  context('Test if create new namespace, export, permissions, clone and delete buttons are not displayed when in create mode', () => {
```

Comment:
why not replace this with manage namespace so all of the buttons are listed here?